### PR TITLE
Add CompliFi skin

### PR DESCRIPTION
**Pull request by valeevbulat**
.complifi {
  --primary-color: #4aa1ff;
  --bg-color: #fdfdfd;
  --text-color: #0c1b32;
  --link-color: #0c1b32;
  --heading-color: #020150;
  --border-color: #e0e0e0;
  --header-bg: #ededf1;
  --block-bg: #f5f5f7;
}

@media (prefers-color-scheme: dark) {
  .complifi {
    --primary-color: #4aa1ff;
    --bg-color: #05142c;
    --text-color: #cccccc;
    --link-color: #ffffff;
    --heading-color: #ffffff;
    --border-color: #2d3a4e;
    --header-bg: #0c1b32;
    --block-bg: #0c1b32;
  }
}

#### Hi! What is your PR about?

- [x] Add or edit a skin
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- ~Add images for your space~ (deprecated) Add the image URL on the space settings directly
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/complifi.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (gokaiorg) [inline skins/complifi.scss:1]: Please type your code in the `.scss` file.
```suggestion
.complifi {
  --primary-color: #4aa1ff;
  --bg-color: #fdfdfd;
  --text-color: #0c1b32;
  --link-color: #0c1b32;
  --heading-color: #20150;
  --border-color: #e0e0e0;
  --header-bg: #ededf1;
  --block-bg: #f5f5f7;
}

@media (prefers-color-scheme: dark) {
  .complifi {
    --primary-color: #4aa1ff;
    --bg-color: #05142c;
    --text-color: #cccccc;
    --link-color: #ffffff;
    --heading-color: #ffffff;
    --border-color: #2d3a4e;
    --header-bg: #0c1b32;
    --block-bg: #0c1b32;
  }
}
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by valeevbulat** [comment]
Corrected the file

**Review by gokaiorg — APPROVED** [review]

**Comment by valeevbulat** [comment]
@gokaiorg Hello! Could you tell me why our skin is not displayed in the settings?

**Comment by gokaiorg** [comment]
> @gokaiorg Hello! Could you tell me why our skin is not displayed in the settings?

@valeevbulat  It may take some time to be deployed. It should be displayed very soon.

**Pull request merged**